404.html

```diff
@@ -5,6 +5,95 @@
   <script type="text/javascript">
 	window.isErrorPage = true;
 	window.errorCode = '404';
+
+	const redirectMap = new Map(
+		[
+			['ae_ar', '/'],
+			['ae_en', '/'],
+			['africa', '/'],
+			['ar', '/es/'],
+			['at', '/de/'],
+			['au','/'],
+			['be_en', '/'],
+			['be_fr/', '/fr/'],
+			['be_nl', '/nl/'],
+			['bg', '/'],
+			['ca', '/'],
+			['ca_fr', '/fr/'],
+			['ch_de', '/de/'],
+			['ch_fr', '/fr/'],
+			['ch_it', '/it/'],
+			['cis_en', '/'],
+			['cis_ru', '/'],
+			['cl', '/es/'],
+			['co', '/es/'],
+			['cy_en', '/'],
+			['cz', '/'],
+			['ee', '/'],
+			['gr_en', '/'],
+			['hk_en', '/'],
+			['hk_zh', '/tw/'],
+			['hu', '/'],
+			['id_en', '/'],
+			['id_id', '/'],
+			['ie', '/'],
+			['il_en', '/'],
+			['il_he', '/'],
+			['in', '/'],
+			['in_hi', '/'],
+			['la', '/es/'],
+			['lt', '/'],
+			['lu_de', '/de/'],
+			['lu_en', '/'],
+			['lu_fr', '/fr/'],
+			['lv', '/'],
+			['mena_ar', '/'],
+			['mena_en', '/'],
+			['mt', '/'],
+			['mx', '/es/'],
+			['my_en', '/'],
+			['my_ms', '/'],
+			['nz', '/'],
+			['pe', '/es/'],
+			['ph_en', '/'],
+			['ph_fil', '/'],
+			['pl', '/'],
+			['pt', '/br/'],
+			['ro', '/'],
+			['ru', '/'],
+			['sa_ar', '/'],
+			['sa_en', '/'],
+			['sea', '/'],
+			['sg', '/'],
+			['si', '/'],
+			['sk', '/'],
+			['th_en', '/'],
+			['th_th', '/'],
+			['tr', '/'],
+			['ua', '/'],
+			['uk', '/'],
+			['vn_en', '/'],
+			['vn_vi', '/'],
+		]	
+	);
+	
+	(function (win, doc) {
+		const { href, pathname } = win.location;	
+
+		let originalPath = pathname.split("/");
+		let userLocale = originalPath[1]; 
+		let redirectMapping = redirectMap.get(userLocale);
+		
+		if (redirectMapping){
+			let localeCookie = userLocale;
+			// making sure just the specific string between the forward slashes are replaced.
+			userLocale = "/"+ userLocale +"/";
```

Comment:
Please use single quote and space left and right to the operators.

Reply:
Actually, rather use string literals to concatenate text:
```suggestion
			userLocale = `"/${userLocale}/`;
```
`npm run lint` should catch this.

Reply:
Done!